sample.env

```diff
@@ -37,6 +37,11 @@ DOCKER_SHARED_VOLUME_WINDOWS=
 ODE_RSU_USERNAME=
 ODE_RSU_PASSWORD=
 
+# Values to sign TIM messages delivered to RSUs/SDX
+# Accepted values are true or false and if not set will default to false
+DATA_SIGNING_ENABLED_RSU=
+DATA_SIGNING_ENABLED_SDW=
```

Comment:
It might be good to add a note to the README as well regarding how signing can be configured.

Reply:
Added a note to the README on how signing can be configured